### PR TITLE
[indexer] set up ingestion test

### PR DESCRIPTION
**Pull request by emmazzz**
## Description 

Adds tests for indexer v2 ingestion and populating of the transactions table.

## Test Plan 

PR is test itself.

---
If your changes are not user-facing and not a breaking change, you can skip the following section. Otherwise, please indicate what changed, and then add to the Release Notes section as highlighted during the release process.

### Type of Change (Check all that apply)

- [ ] protocol change
- [ ] user-visible impact
- [ ] breaking change for a client SDKs
- [ ] breaking change for FNs (FN binary must upgrade)
- [ ] breaking change for validators or node operators (must upgrade binaries)
- [ ] breaking change for on-chain data layout
- [ ] necessitate either a data wipe or data migration

### CODE REVIEW

**Review by wlmyng — APPROVED** [review]

**Review by sadhansood — APPROVED** [review]

File: crates/sui-indexer/src/schema_v2.rs [hunk 1]
```diff
@@ -341,8 +325,4 @@ diesel::allow_tables_to_appear_in_same_query!(
     tx_input_objects,
     tx_recipients,
     tx_senders,
-    tx_indices,
 );
-
-use diesel::sql_types::Text;
```
Comment (gegaowp) [inline crates/sui-indexer/src/schema_v2.rs:347]: @oxade is `query_cost` necessary here? This PR removed it but passed all CI tests.

Comment (oxade) [reply crates/sui-indexer/src/schema_v2.rs:347]: We disabled it in code so no test triggers it.
But yeah I wouldn't remove it just yet. Let minimize potential DB conflicts.

Comment (wlmyng) [reply crates/sui-indexer/src/schema_v2.rs:347]: We don't use the custom sql fn query_cost anymore, so we can consider removing it from our test code instead

Comment (gegaowp) [reply crates/sui-indexer/src/schema_v2.rs:347]: I am not quite following here, but feel free to add it back if it is necessary

**Pull request merged**